app/src/main/res/raw/sessions_ja.json

```diff
@@ -812,7 +812,7 @@
       "id": 4,
       "name": "ルームC"
     },
-    "language_id": "",
+    "language_id": "ja",
```

Comment:
Yes, I speak in Japanese. But I am going to try to speech in English at only first greeting.